src/tqec/templates/orchestrator.py

```diff
@@ -48,7 +49,7 @@ def get_corner_position(
     )
 
 
-class TemplateOrchestrator(JSONEncodable):
+class TemplateOrchestrator(Template):
```

Comment:
Now the `TemplateOrchestrator` is a `Template`, maybe a name like `ComposedTemplate` is better.

Reply:
I need to think a little bit more about that. My initial opinion is that `ComposedTemplate` is too generic and not descriptive enough. 
The example I have in mind is the soon-to-be-included `TemplateStack` (see in #100). In my mind, it is a "composed" template in the sense that it is a composition of templates, but will not be a subclass of `TemplateOrchestrator`.
I am fine with `TemplateOrchestrator` staying, but I understand that it might be more meaningful to have a name that finishes with `Template` to make the inheritance obvious.

Reply:
A few ideas for names:
1. `PlanarTemplate` (miss the "several templates together" part),
2. `TemplateOrchestratorTemplate` (in my opinion, not clearer than `TemplateOrchestrator` but longer),
3. `ComposedTemplate` (with my opinion above),
4. `TemplateOrchestrator` (not ending in "Template", which could help, if present, making the fact that it is a `Template` sub-class clearer).

Any feelings / other proposition?

Reply:
My thoughts on the naming hierarchy are as follows:

- `Template`: serves as the foundational class.
- `ComposedTemplate`: represents a template that is constructed from multiple non-overlapping templates arranged on a 2D plane.
- `StackedTemplate`: denotes a template that originates from several templates layered in 3D space, though it is designed to be viewed solely from a top-down perspective to project it onto 2D plane.

Once the docstrings for the classes are clearly written, confusion should be minimal. However, the decision rests with you, as I might not fully grasp the intentions behind your original design.

Reply:
> However, the decision rests with you, as I might not fully grasp the intentions behind your original design.

Original design is not as important as current design, and your opinion is, in my opinion, worth more than mine here as you are not biased by the original design decisions (and I am).

Let's go with `ComposedTemplate` then. If my points above did not make you change your mind then it means they are not strong enough to keep the original name.